evidence/liamaharon/0001-2024-05-evidence.md

```diff
@@ -0,0 +1,63 @@
+- End date: 10/05/2024
+
+## Evidence
+
+| Areas of Contribution | Tasks  | Links   | Notes   |
+|---|---|---|---|
+| Polkadot SDK | Stop Balances pallet erroneously double incrementing and decrementing consumers | [SDK 1976](https://github.com/paritytech/polkadot-sdk/pull/1976) |  |
+| Polkadot SDK | Allow Locks/Holds/Reserves/Freezes by default when using pallet_balances TestDefaultConfig | [SDK 1880](https://github.com/paritytech/polkadot-sdk/pull/1880) |   |
+| Polkadot SDK | Optimise profile dev wasm builds | [SDK 1851](https://github.com/paritytech/polkadot-sdk/pull/1851) |   |
+| Polkadot SDK | Remove dmp_queue pallet from Westend SP runtimes | [SDK 2516](https://github.com/paritytech/polkadot-sdk/pull/2516) |   |
+| Polkadot SDK | Set frame_system::LastRuntimeUpgrade after running try-runtime migrations | [SDK 2515](https://github.com/paritytech/polkadot-sdk/pull/2515) |   |
+| Polkadot SDK | Delete undecodable Westend Asset Hub Balances::Hold and Nfts::ItemMetadataOf | [SDK 2309](https://github.com/paritytech/polkadot-sdk/pull/2309) |   |
+| Polkadot SDK | Migrate fee payment from Currency to fungible | [SDK 2292](https://github.com/paritytech/polkadot-sdk/pull/2292) |   |
+| Polkadot SDK | Improve VersionedMigration naming conventions | [SDK 2264](https://github.com/paritytech/polkadot-sdk/pull/2264) |   |
+| Polkadot SDK | Tokens in FRAME Docs | [SDK 2802](https://github.com/paritytech/polkadot-sdk/pull/2802) |   |
+| Polkadot SDK | Development Environment Advice Reference Doc | [SDK 2759](https://github.com/paritytech/polkadot-sdk/pull/2759) |   |
+| Polkadot SDK | Improve TryDecodeEntireState output | [SDK 2724](https://github.com/paritytech/polkadot-sdk/pull/2724) |   |
+| Polkadot SDK | chain-spec-builder: Improve output path example | [SDK 2693](https://github.com/paritytech/polkadot-sdk/pull/2693) |   |
+| Polkadot SDK | Finish documenting #[pallet::xxx] macros | [SDK 2638](https://github.com/paritytech/polkadot-sdk/pull/2638) |   |
+| Polkadot SDK | Sync Cargo.toml and crates.io versions | [SDK 3034](https://github.com/paritytech/polkadot-sdk/pull/3034) |   |
+| Polkadot SDK | Update translate to use defensive! | [SDK 2985](https://github.com/paritytech/polkadot-sdk/pull/2985) |   |
+| Polkadot SDK | Introduce storage attr macro #[disable_try_decode_storage] | [SDK 3454](https://github.com/paritytech/polkadot-sdk/pull/3454) |   |
+| Polkadot SDK | pallet-uniques: decrement total_deposit when clearing collection metadata | [SDK 3976](https://github.com/paritytech/polkadot-sdk/pull/3976) |   |
+| Polkadot SDK | remote-externalities: retry get child keys query | [SDK 4280](https://github.com/paritytech/polkadot-sdk/pull/4280) |   |
+| Fellowship Runtimes | Fix migrations and add migration CI checks | [RUNTIMES 75](https://github.com/polkadot-fellows/runtimes/pull/75) |   |
+| Fellowship Runtimes | Fix migrations and add migration CI checks | [RUNTIMES 75](https://github.com/polkadot-fellows/runtimes/pull/75) |   |
```

Comment:
This one is mentioned twice.

Btw did you auto-generate or write it manually? I used some yaml->md generator.